drivers/serial/uart_cdns_registers.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Meta Platforms, Inc. and its affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <stdint.h>
```

Comment:
imo: it would be cleaner from a file tree to have both of these headers just at the top of the `uart_cdns.c` file

Reply:
Will keep it as is for now. Thanks.

Reply:
Please condense down to a single .h file and .c file or just a single .c file. I don't believe having 3 files is necessary